### DISCUSSION TITLE
How do you decode Message payload?

### DISCUSSION THREAD

**Discussion by ShengMTsai** [dotnet/MQTTnet#1462] [General]
Hi, is there any example on how to decode the Message Payload (byte array)? The content type is "application/sparkplug".

**Comment by chkr1011** [comment]
Handling the payload is not part of this project. So you must check the RFC of the content type and find c# libraries to decode it properly.

**Comment by SeppPenner** [comment]
> Hi, is there any example on how to decode the Message Payload (byte array)? The content type is "application/sparkplug".

You can take a look at https://github.com/SeppPenner/SparkplugNet, there should be examples. As @chkr1011 told, this is not part of the core MQTTnet project.
